WcaOnRails/config/unicorn.rb

```diff
@@ -13,16 +13,15 @@
   raise "Unrecognized RACK_ENV: #{rack_env}, must be one of #{allowed_environments.keys.join ', '}"
 end
 if rack_env == "development"
-  listen 3000
   worker_processes 1
 else
-  stderr_path "#{dir}/log/unicorn-#{rack_env}.log"
-  stdout_path "#{dir}/log/unicorn-#{rack_env}.log"
+  stderr_path $stdout
```

Comment:
Are you absolutely certain that setting this to `$stderr` will make the logs invisible to CloudWatch?

Reply:
Setting it to $stderr also works. Just not to the file